rootfs/etc/services.d/home-assistant/run

```diff
@@ -2,9 +2,13 @@
 # ==============================================================================
 # Start Home Assistant service
 # ==============================================================================
+EXCLUDE_HARDWARE_LIST="Annapurna Labs Alpine AL314|Other Excluded Hardware"
```

Comment:
Please use bash arrays for this instead, it would also remove the need to use egrep later on.

Reply:
Another thought, maybe make it an ENV setting? That way anyone who wants to disable it, can disable it and removes the need to maintain such a list.

Reply:
I think using ENV flag is easier, thank you for suggestion.